### DISCUSSION TITLE
How about a Date + Time Picker ?

### DISCUSSION THREAD

**Discussion by dukesx** [mantinedev/mantine#238] [Feedback]
Hi,

It's wonderful to see so many awesome people help build this wonderful library. Now i was hoping if someone is willing to work on this component as well ? Generally you can use separate month and time component but then nowdays "date-time" picker is such a common finding that at least i am accustomed to using it. 

Much more convenient when it comes for things like Appointments ? I do run a clinic as well, and its super useful to allow people the convenience of selecting a day and time together, which is suitable and available. I mean, no date is complete without time 😁

Its just a suggestion, looking forward to feedback.

**Comment by rtivital** [comment]
@dukesx please create an issue for this, please also provide a design reference that should be used when building a component

**Reply by dukesx** [reply]
Sure absolutely
